sdb/commands/zfs/zio.py

```diff
@@ -124,6 +124,14 @@ def from_zio(self, zio: drgn.Object) -> Iterable[drgn.Object]:
                 yield from self.from_zio(c.zl_parent)
         self.level -= 1
 
+    @staticmethod
+    def zio_has_parents(zio: drgn.Object) -> bool:
+        parent_list = zio.io_parent_list.list_head.address_of_()
+        first_parent = parent_list.next
+        if parent_list != first_parent:
```

Comment:
[nit] this could be simplified to `return parent_list.next != parent_list`, I think.

Reply:
I tried doing that but `mypy` was complaining that it's not `bool` type :/ so I changed it to this